Development

Shamanou opened the pull request:
enabled python scripting in rdf skeleton custom fields and added a vairable called baseUri which can be called in that interface.

Shamanou committed:
Added python as a option for custom expressions in the in the rdf semantic skeleton menu

Shamanou committed:
Fixed minor issues with python scripting in rdf skeleton

Shamanou committed:
Made base uri a variable which can be used in the python scripting interface of the rdf skeleton

Shamanou committed:
Added tests for python scripting in rdf skeleton and baseuri variable check

kburger reviewed (commented):
What is the difference between 'extensions/fairifier-rdf-extension/scripts/rdf-data-table-view.js' and 'module/scripts/rdf-data-table-view.js'? The code and the applied changes seem duplicated.

Shamanou commented:
The fairifier extensions patches scripts by copying them over existing scripts in other directories, so the  code is edited in the extension and on build copied over the other scripts. Some scripts i put in the fairifier extension and in the ant build script without changing them. So i dont have to keep updating the ant  script in the future.

Shamanou committed:
Bugfixes

- fixed ui bug in posting to fdp 
- added error messages to fdp retrieval wich are viewable in front end
- shortened entries in option fields in fdp datasets and catalogs

Shamanou committed:
Fixed bug where adding ontologies from uri did not work

Shamanou committed:
fixed bug where filling base uri of prefix prevented selecting the base uri in the file

Shamanou committed:
Fix multiple bugs relating to prefixes and added a prefix test

Shamanou committed:
Fixed bug where only rdf:label worked in reconciliation

Shamanou committed:
rdfxml parsing in file format detection now works
